test/unit/Controllers/NodeController.test.ts

```diff
@@ -0,0 +1,39 @@
+import sinon from "sinon";
+import { expect } from "chai";
+import { Request, Response } from "express";
+import { NodeService } from "../../../src/Services/NodeService";
+import { NodeController } from "../../../src/Controller/Api/NodeController";
+import { fail } from "assert";
+
+describe("NodeController", function () {
+    describe('POST /user/node', () => {
+        const nodeServiceStub = sinon.createStubInstance(NodeController);
+        nodeServiceStub.createNode.resolves({ url: 'some url', token: 'some token', address: 'some address' });
+
+        it('should add new node to the database', async function () {
+            // try {
```

Comment:
this try/catch is required for the test because if something fails in the test and is not catched, the test could be displayed as PASS.